File: src/UnitTestsCS/UnitTest1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Microsoft.VisualStudio.TestPlatform.UnitTestFramework;
using GLRT;
namespace UnitTestsCS
{
    [TestClass]
    public class UnitTest1
    {
        [TestMethod]
        public void TestMethod1()
        {
            Mesh mesh1 = new Mesh();
            Assert.AreEqual(43, mesh1.GetMagicNumber());
        }
    }
}

File: src/UnitTestsCX/unittest1.cpp
#include "pch.h"
#include "CppUnitTest.h"

using namespace Microsoft::VisualStudio::CppUnitTestFramework;
using namespace GLRT;

namespace UnitTestsCX
{
    TEST_CLASS(UnitTestCX)
    {
    public:
        TEST_METHOD(TestMethod1)
        {
            // TODO: Your test code here
			Mesh ^ mesh = ref new Mesh();
			Assert::AreEqual(43, mesh->GetMagicNumber());
        }
    };
}